README.md

```diff
@@ -31,6 +31,7 @@ Helminator takes the following environment variables:
 |Variable|Description|Example|
 |:--------|:-----------|:-------|
 |`HELMINATOR_ROOT_DIR`|Directory to scan|`/path/to/playbook`|
+|`HELMINATOR_ENABLE_PRERELEASES`|Configure Helminator to process prereleases|`true` or `false`|
```

Comment:
`HELMINATOR_ENABLE_PRERELEASES`|Configure Helminator to process prereleases|`true` or `false` (default: `false`) |

Reply:
done